api/utils/error_parsing.ts

```diff
@@ -75,7 +75,9 @@ export function parseBigqueryEvalError(error: IBigqueryEvaluationError) {
     // extract everything after the very last [ in the string
     const bracketsString = error.message.split("[").slice(-1)[0];
     const [_, lineNumber, columnNumber] = bracketsString.match(/([0-9]*)[^0-9]*([0-9]*).*/);
-    evalError.errorLocation = { line: Number(lineNumber), column: Number(columnNumber) };
+    if (lineNumber && columnNumber) {
```

Comment:
this needs fixing for errors at `(0, 0)`. probably need to check the match status from the above line.